postgres/README.md

```diff
@@ -4,10 +4,9 @@
 
 ## Overview
 
-Get metrics from PostgreSQL in real time to:
+The Postgres integration provides health and performance metrics for your Postgres database in real time. Visualize these metrics with the included dashboard and create monitors to alert your team on PostgreSQL states.
```

Comment:
is it truly real time? if not, we should say "near real-time"

"with the included dashboard" sounds strange also -- maybe we should say with the "OOTB" dashboard or "provided" dashboard?